internal/verification/phaverify.go

```diff
@@ -94,8 +102,19 @@ func (v *Verifier) VerifyDiagnosisCertificate(ctx context.Context, authApp *aamo
 			return nil, err
 		}
 
+		if cacheVal == nil {
```

Comment:
This should never happen? I don't think we cache nil values.

Reply:
See line 93 - the primary lookup function can return `nil` as a valid cached value now
the in memory cache implementation allows it.